Competition2017/src/competition/subsystems/climbing/ClimbingSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package competition.subsystems.climbing;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.CANTalon.TalonControlMode;
+
+import com.google.inject.Inject;
+
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+
+import xbot.common.controls.actuators.XCANTalon;
+
+import xbot.common.injection.wpi_factories.WPIFactory;
+
+import xbot.common.properties.DoubleProperty;
+
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class ClimbingSubsystem extends BaseSubsystem {
+
+    private static Logger log = Logger.getLogger(ClimbingSubsystem.class);
+
+    private final DoubleProperty climbPowerProperty;
+    private final DoubleProperty descendPowerProperty;
+    private final XCANTalon climbingMotor;
+
+    @Inject
+    public ClimbingSubsystem(WPIFactory factory, XPropertyManager propManager){
+
+        log.info("Creating Collector Subsystem");
+
+        climbingMotor = factory.getCANTalonSpeedController(5);
+        climbingMotor.setBrakeEnableDuringNeutral(false);
```

Comment:
I think this might be an instance where we want to enable breaking. That's only if we are using a CIM though.